q2_fondue/entrezpy_clients/_pipelines.py

```diff
@@ -34,10 +34,6 @@ def _get_run_ids(
     Returns:
         list: Run IDs associated with provided ids.
     """
-    nb_runs = get_run_id_count(email, n_jobs, ids, log_level)
-    # source for rounding up to next 1k: https://stackoverflow.com/a/8866125
-    retmax = int(math.ceil(nb_runs / 10000.0)) * 10000
```

Comment:
There is no more need to count anything before we execute any requests. `entrezpy` can do it for us.

Reply:
byebye nice rounding 👋🏼